x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/use_alert_suppression.test.tsx

```diff
@@ -16,51 +16,15 @@ describe('useAlertSuppression', () => {
       .mockReturnValue(false);
   });
 
-  it('should return isSuppressionEnabled false if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is disabled', () => {
-    const { result } = renderHook(() => useAlertSuppression('new_terms'));
-
-    expect(result.current.isSuppressionEnabled).toBe(false);
-  });
-
-  it('should return isSuppressionEnabled true if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is enabled', () => {
-    jest
-      .spyOn(useIsExperimentalFeatureEnabledMock, 'useIsExperimentalFeatureEnabled')
-      .mockReset()
-      .mockReturnValue(true);
-    const { result } = renderHook(() => useAlertSuppression('new_terms'));
-
-    expect(result.current.isSuppressionEnabled).toBe(true);
-  });
-
-  it('should return the correct isSuppressionEnabled value fot threat_match rule type', () => {
-    const { result } = renderHook(() => useAlertSuppression('threat_match'));
-
-    expect(result.current.isSuppressionEnabled).toBe(true);
-  });
-
-  describe('Eql Rule', () => {
-    it('should return the correct isSuppressionEnabled value if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is enabled', () => {
-      jest
-        .spyOn(useIsExperimentalFeatureEnabledMock, 'useIsExperimentalFeatureEnabled')
-        .mockReset()
-        .mockReturnValue(true);
-      const { result } = renderHook(() => useAlertSuppression('eql'));
-
-      expect(result.current.isSuppressionEnabled).toBe(true);
-    });
-
-    it('should return the correct isSuppressionEnabled value if rule Type exists in SUPPRESSIBLE_ALERT_RULES and Feature Flag is disabled', () => {
-      const { result } = renderHook(() => useAlertSuppression('eql'));
-
-      expect(result.current.isSuppressionEnabled).toBe(false);
-    });
-  });
-
-  it('should return the correct isSuppressionEnabled value if rule Type exists in SUPPRESSIBLE_ALERT_RULES', () => {
-    const { result } = renderHook(() => useAlertSuppression('query'));
-
-    expect(result.current.isSuppressionEnabled).toBe(true);
-  });
+  (['new_terms', 'threat_match', 'saved_query', 'query', 'threshold'] as Type[]).forEach(
+    (ruleType) => {
+      it(`should return the isSuppressionEnabled true for ${ruleType} rule type that exists in SUPPRESSIBLE_ALERT_RULES`, () => {
```

Comment:
Nit: besides the "should" (😉), this mainly describes the implementation. I would instead focus on the behavior:

```suggestion
      it(`isSuppressionEnabled is true for the "${ruleType}" rule type`, () => {
```